README.md

```diff
@@ -1,140 +1,127 @@
+This template is for any HM Government analyst(s) who codes! It's main purpose is to:
 
-To create a new repository structure from this cookiecutter, install the required Python packages via `pip` by running
-the following command:
+1. Provide a lightweight, Agile-like approach to Analytical Quality Assurance (AQA)
+2. Help quickly set up standardised project structures.
 
-```shell script
-make requirements
-```
-
-To make developments to this project, install the development Python packages via `pip` using:
-
-```shell script
-make requirements-dev
-```
-
-Once you have installed the packages, remember to [set up pre-commit hooks](#installing-pre-commit-hooks).
-
-### Creating a Python virtual environment
+For more information about AQA, see [The Aqua Book][aqua-book], and its [resources][aqua-book-resources].
 
-Creating a Python virtual environment depends on whether you are using [base Python](#base-python-interpreter) or
-[Anaconda](#anaconda-interpreter) as your interpreter.
+Rather than using different files and locations to store AQA and documentation, our intention is to centralise as much
+of this as possible onto your Git repository hosting platform, e.g. GitHub or GitLab.
 
-#### Base Python interpreter
+We use nudges, such as checklists in pull/merge requests, to minimise the burden on contributors and reviewers to
+complete AQA checks. This results in faster iterative development and deployment, whilst ensuring HM Government-wide
+standards on assurance are met.
 
-If you are using base Python, there are multiple ways to create virtual environments in Python using `pip`, including
-(but not limited to):
+We have also included [pre-commit hooks][pre-commit] to prevent accidental committing of secrets, large data files, and
+Jupyter notebook outputs for security purposes.
 
-- [`venv`](https://docs.python.org/3/tutorial/venv.html);
-- [`virtualenv`](https://virtualenv.pypa.io/en/stable/);
-- [`pipenv`](https://github.com/pypa/pipenv); and
-- [`pyenv`](https://github.com/pyenv/pyenv) with its `virtualenv` [plugin](https://github.com/pyenv/pyenv-virtualenv).
+## Getting started
 
-Follow the documentation of your chosen method to create a Python virtual environment.
+> ⚠️ Only Unix-based systems (macOS, Linux, ...), and Python projects for GitHub or GitLab are supported — feel free to
+> [contribute](#contributing) to support other operating systems/programming languages!
 
-#### Anaconda interpreter
+To use this template to start your next coding project, make sure your system meets the [requirements](#requirements).
 
-If you are using [Anaconda or `conda`](https://www.anaconda.com/), following their
-[documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) to set up a
-conda environment.
+Once you're all set up, open your terminal, navigate to the directory where you want your new repository to exist, and
+run the following commands:
```

Comment:
Can we be explicit and mention the below? (Sorry if I mentioned this before and you said it wasn't a good idea, I missed the comment!)
```suggestion
run the following commands in your shell/terminal:
```